community.json

```diff
@@ -1730,6 +1730,13 @@
         "revision": "795449cb43ccdba949c814746d6ea90eee0a7259",
         "state": "working",
         "url": "https://github.com/aymhce/yunohost_ynh"
+    },
+     "zabbix": {
+        "branch": "master",
+        "level": 2,
```

Comment:
Please don't specify the level manually, it's meant to be the result of automatic test by the CI

Reply:
Sorry, who make this test ? me ? you ? using https://github.com/YunoHost/package_check ?

Reply:
Nobody. It's the CI who did automatic tests : https://ci-apps.yunohost.org/ci/

Reply:
You have just to remove this line.